widgetssdk/src/main/java/com/glia/widgets/StringProviderImpl.kt

```diff
@@ -6,8 +6,8 @@ class StringProviderImpl(private val resourceProvider: ResourceProvider): String
 
     override fun getRemoteString(stringKey: Int, vararg values: String): String {
         // TODO IMPLEMENT NEW METHOD FROM CORE
-        val fallback = resourceProvider.getString(stringKey, values)
         val key = resourceProvider.getResourceKey(stringKey)
+        val fallback = resourceProvider.getString(stringKey, *values)
```

Comment:
I don't get the idea of the fallback.

Reply:
of fallback or of the change in the fallback?

Fallback is the native string value that will be used ONLY if no value is present in the remote strings